config/install/field.storage.node.field_hyperdrive_rating.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_hyperdrive_rating
+field_name: field_hyperdrive_rating
+entity_type: node
+type: integer
```

Comment:
This field should be "Number (decimal)" type.
![starships_field_hyperdrive_rating_type](https://github.com/notS0Solo/swapimigration/assets/1434979/21c08d7f-19eb-47b3-af5c-68ef445c2c6d)

Reply:
Fixed